app/backend/src/couchers/notifications/render.py

```diff
@@ -862,6 +863,8 @@ def render_notification(user, notification) -> RenderedNotification:
             email_template_args={
                 "title": title,
                 "message": message,
+                "donation_message": donation_message,
+                "donation_link": f"{urls.donation_url()}?source=strong-verification-email",
```

Comment:
This needs to be a separate email template

Reply:
Okay I made a separate one and switched it to use that